packages/shorebird_code_push_protocol/lib/src/models/release_artifact.g.dart

```diff
@@ -14,13 +14,13 @@ ReleaseArtifact _$ReleaseArtifactFromJson(Map<String, dynamic> json) =>
       json,
       ($checkedConvert) {
         final val = ReleaseArtifact(
-          id: $checkedConvert('id', (v) => v as int),
-          releaseId: $checkedConvert('release_id', (v) => v as int),
+          id: $checkedConvert('id', (v) => (v as num).toInt()),
```

Comment:
Were all of these other .g.dart changes intentional? Are you using a newer version of `build_runner`?

Reply:
the build runner version is configured in the project pubspec, but we don't have the lock file under version control, so I am pretty sure I ended up using a newer version than the last time that the generated files were created.

maybe it would be better if I extract these changes to its own separate PR?

Reply:
Yeah I think it's fine either way just wanted to double check.

Reply:
extracted to its own PR: https://github.com/shorebirdtech/shorebird/pull/1994